framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -244,15 +244,23 @@
     protected final BaseTypeChecker checker;
 
     /**
-     * Map from class name (canonical name) of an annotation, to the annotation in the Checker
-     * Framework that will be used in its place.
+     * Map from class names of aliases, to the annotations in the Checker Framework that will be
```

Comment:
Does class name here mean fully-qualified name? There's a good explanation of the various kinds of javac names here: https://checkerframework.org/manual/#signature-checker

Reply:
I think @wmdietl and I shared the same understanding here on the names:
* Alias means the annotation to be replaced when running the checker framework. For example, `@NotNull` is an alias of `@NonNull` in checker framework. `@IndexFor` is an alias of the combination of `@NonNegative` and `@LTLengthOf`. 
* Canonical means the annotation to replace with when running the checker framework. For example `@NonNull` (`@org.checkerframework.checker.nullness.qual.NonNull`) is the canonical annotation for all kinds of `@NonNull`s in other packages.

Correct me if my understanding is incorrect.

Reply:
This discussion suggests that  "canonical name" is a misleading term.  "Name of the canonical annotation" would be more accurate.
Perhaps it's worth a phrase to define "canonical annotation" -- I'm not sure about that.

Even after clarifying that issue, the documantation should indicate the string format of the class name:  whether it is a fully-qualified name, or a simple name, or something else.  That would address the code review comment.